python/mxnet/contrib/onnx/_import/import_onnx.py

```diff
@@ -110,6 +110,10 @@ def from_onnx(self, graph):
                 self._nodes[name_input] = symbol.Variable(name=name_input)
                 self._renames[i.name] = name_input
 
+        # For storing arg  and aux params for the graph.
+        auxDict = {}
+        argDict = {}
```

Comment:
Why not initiate these variables in the constructor

Reply:
We don' t need class variables . Local variables should suffice.